src/calculation/DetailedResultsBuilder.ts

```diff
@@ -320,6 +330,39 @@ export function createPatientPopulationValues(
   };
 }
 
+export function handleStratificationValues(
+  populationGroup: fhir4.MeasureGroup,
+  populationResults: PopulationResult[],
+  stratifierResults: StratifierResult[]
+): StratifierResult[] {
+  if (populationGroup.stratifier) {
+    populationGroup.stratifier.forEach(strata => {
```

Comment:
You may want to iterate over the stratifierResults instead of the group's stratifiers and do the lookup the opposite way around similar to how it is done in the MeasureReport builder. https://github.com/projecttacoma/fqm-execution/blob/0b752a1a5da6a3aa5ee4d1c5a5df50b292ff5fb5/src/calculation/MeasureReportBuilder.ts#L237
The code there also handles to two different ways the stratifiers are referenced via id or codes.